prometheus/exporter/watchdog.py

```diff
@@ -321,14 +323,26 @@ def main(argv):
     address = argv[3]
     
     # init the logger
+    loggerCommon.propagate = False
     logger.setLevel(logging.INFO)  
     logger.propagate = False
-    fileHandler = RotatingFileHandler(logDir + "/watchdog.log", maxBytes= 1024 * 1024 * 1, backupCount=5)  
+    fileHandler = RotatingFileHandler(logDir + "/watchdog.log", maxBytes= 1024 * 1024 * 100, backupCount=5)  
     fileHandler.setLevel(logging.INFO)  
     formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")  
     fileHandler.setFormatter(formatter)  
-    logger.addHandler(fileHandler)  
-
+    logger.addHandler(fileHandler) 
+
+    loggerRoot.setLevel(logging.INFO)  
+    loggerRoot.propagate = False
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.INFO)
+    # create formatter and add it to the handlers
+    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    ch.setFormatter(formatter)
+    loggerRoot.addHandler(ch) 
+    loggerRoot.info("Please view log trace:") 
+    loggerRoot.info("Log trace path {}/watchdog.log".format(logDir))
+    
```

Comment:
in loggerRoot, please explain why you do not out log in k8s log directly. 
also, please output the current configuration of logger (e.g.,, maxBytes, etc.)